src/Artsy/Relay/renderWithLoadProgress.tsx

```diff
@@ -17,56 +17,45 @@ const SpinnerContainer = styled.figure`
 
 export const LoadingClassName = "relay-loading"
 
+const handleError = error => {
+  // In tests we want errors to clearly bubble up.
+  if (typeof jest !== "undefined") {
+    throw error
+  }
+
+  if (error.message) {
+    console.error(error.message)
+  }
+
+  const networkError = error as any
+  if (networkError.response && networkError.response._bodyInit) {
+    let data = networkError.response._bodyInit
+    if (data) {
+      try {
+        data = JSON.parse(data)
+        console.error(`Metaphysics Error data:`, data)
```

Comment:
```js
`import { createLogger } from 'Utils/logger'`

const logger = createLogger('Artsy/Relay/renderWithLoadProgress')
logger.error(data) 
```

Reply:
Wasn't planning on addressing this, but since you did all the work for me, I got this in with da360a3. ❤️ 🏆 💯